src/cli/commands/audit.js

```diff
@@ -145,6 +146,7 @@ export async function run(config: Config, reporter: Reporter, flags: Object, arg
   const audit = new Audit(config, reporter, {
     groups: flags.groups || OWNED_DEPENDENCY_TYPES,
     level: flags.level || DEFAULT_LOG_LEVEL,
+    registry: flags.registry,
```

Comment:
Why don't you add `|| YARN_REGISTRY` here instead of in the `_fetchAudit` method?
Here it'll look more consistent, in my opinion.